test/manifolds/test_sphere.jl

```diff
@@ -0,0 +1,356 @@
+            for k in [1, -1]
+                p *= k
+                i = Manifolds.get_chart_index(M, A, p)
+                @test i === (p[1] < 0 ? :south : :north)
+                a = get_parameters(M, A, i, p)
+                q = get_point(M, A, i, a)
+                @test isapprox(M, p, q)
+
+                p2 = [1 / sqrt(2), 1 / sqrt(2), 0]; p3 = [0, 1 / sqrt(2), 1 / sqrt(2)]
+                X2 = log(M, p, p2); X3 = log(M, p, p3)
+                B = induced_basis(M, A, i, Manifolds.TangentSpaceType())
+
+                X2B = get_coordinates(M, p, X2, B)
+                X3B = get_coordinates(M, p, X3, B)
+
+                @test inner(M, p, X2, X3) ≈ dot(X2B, local_metric(M, p, B) * X3B)
+
+                X2back = get_vector(M, p, X2B, B)
+                X3back = get_vector(M, p, X3B, B)
+                @test isapprox(M, p, X2, X2back)
+                @test isapprox(M, p, X3, X3back)
+
+                @testset "transition_map" begin
+                    other_chart = i === :south ? :north : :south
+                    a_other = Manifolds.transition_map(M, A, i, other_chart, a)
+                    @test isapprox(M, p, get_point(M, A, other_chart, a_other))
+
+                    a_other2 = allocate(a_other)
+                    Manifolds.transition_map!(M, a_other2, A, i, A, other_chart, a)
+                    @test isapprox(M, p, get_point(M, A, other_chart, a_other2))
+
+                    a_other2 = allocate(a_other)
+                    Manifolds.transition_map!(M, a_other2, A, i, other_chart, a)
+                    @test isapprox(M, p, get_point(M, A, other_chart, a_other2))
+                end
+            end
+            # TODO: test RetractionAtlas somewhere
+        end
+
+        @testset "sectional curvature" begin
+            M = Sphere(2; parameter = :field)
+            K = Manifolds.sectional_curvature_matrix(
+                M,
+                [1.0, 0.0, 0.0],
+                DefaultOrthonormalBasis(),
+            )
+            @test isapprox(K, [0 1; 1 0])
+            @test isapprox(
+                Manifolds.estimated_sectional_curvature_matrix(
+                    M,
+                    [1.0, 0.0, 0.0],
+                    DefaultOrthonormalBasis(),
+                ),
+                [0.0 1.0; 1.0 0.0],
+                atol = 0.15,
+            )
+        end
+
+        @testset "Local Metric" begin
+            p = [1.0, 0.0, 0.0]
+            for M in [Sphere(2), Sphere(2; parameter = :field)]
+                @test local_metric(M, p, DefaultOrthonormalBasis()) == Diagonal([1.0, 1.0])
+            end
+        end
+
+        #= Add the following to the test suite as functions
+            @testset "Weingarten" begin
+                M = Sphere(2)
+                p = [1.0, 0.0, 0.0]
+                X = [0.0, 2.0, 0.0]
+                V = [0.3, 0.0, 0.0]
+                Y = -X * (p'V)
+                @test Weingarten(M, p, X, V) == Y
+            end
+        =#
```

Comment:
This should be covered, did I forget to remove this? Weingarten is tested I hope.